### ISSUE TITLE
SoundMixer errors (OpenFL 9.3.4)

### ISSUE THREAD

**Issue by hoangdung-qbt** [openfl/openfl#2739]
**Describe the bug**
When calling SoundMixer.stopAll(), I have this issue 

_"Called from openfl/media/SoundMixer.hx line 176"_

**To Reproduce**
1. Playing Sounds 
2. Create a simple button
3. Calling  SoundMixer.stopAll()

**Expected behavior**
_"Called from openfl/media/SoundMixer.hx line 176"_

```
public static function stopAll():Void
	{
		var i = __soundChannels.length;
		while (i > 0)
		{
			__soundChannels[i].stop();
			i--;
		}
	}
```

**OpenFL Targets**
OpenFL 9.3.4
Solution: switch back using OpenFL 9.3.3

All my versions: OpenFL 9.3.4. Lime 8.1.3, HXCPP 4.3.2

**Comment by DetectiveBaldi** [comment]
i believe this is fixed in the latest git commits
more info at #2736

**Comment by joshtynjala** [comment]
> i believe this is fixed in the latest git commits
> more info at https://github.com/openfl/openfl/pull/2736

I believe that's right. The fix will be included in OpenFL 9.4, which I hope to get released within the next month.
